lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -781,15 +781,21 @@ def cmd_transport(*args)
     end
   end
 
+  @@migrate_opts = Rex::Parser::Arguments.new(
+    '-p'  => [true,  'Writable path (eg. /tmp).'],
```

Comment:
This flag is only meaningful on Linux. I would add a note to that effect in the help string.